app/src/main/java/it/niedermann/owncloud/notes/shared/util/NoteUtil.java

```diff
@@ -69,9 +69,11 @@ public static String generateNoteExcerpt(@NonNull String content, @Nullable Stri
             return "";
         }
         if (!TextUtils.isEmpty(title)) {
-            final String trimmedTitle = removeMarkdown(replaceCheckboxesWithEmojis(title.trim()));
-            if (content.startsWith(trimmedTitle)) {
-                content = content.substring(trimmedTitle.length());
+            if (title != null) {
```

Comment:
Please remove this `if`, it is already covered by `!TextUtils.isEmpty(title)` - actually you can just revert the changes in `NoteUtil.java`.